hera_mc/cm_hookup.py

```diff
@@ -62,28 +65,23 @@ def get_hookup(self, hpn, rev, port, at_date, state_args, exact_match=False):
                                                at_date=self.at_date,
                                                exact_match=exact_match)
         hookup_dict = {'hookup': {}}
-        col_len_max = [0, '-']
-
         pols_to_do = self.__get_pols_to_do(hpn, port, check_pol=False)
 
         for k, part in parts.iteritems():
             if not cm_utils._is_active(self.at_date, part['part'].start_date, part['part'].stop_date):
                 continue
-            # if len(part['connections']['ordered-pairs'][0]) == 0:
-            #    continue
             hookup_dict['hookup'][k] = {}
             for pol in pols_to_do:
                 hookup_dict['hookup'][k][pol] = self.__follow_hookup_stream(part['part'].hpn, part['part'].hpn_rev, pol)
-                if len(hookup_dict['hookup'][k][pol]) > col_len_max[0]:
-                    col_len_max[0] = len(hookup_dict['hookup'][k][pol])
-                    col_len_max[1] = k
-        hookup_dict = self.__add_hookup_timing(hookup_dict)
+        if len(hookup_dict['hookup'].keys()) > 0:
+            hookup_dict = self.__add_hookup_timing(hookup_dict)
+            hookup_dict['columns'] = self.__get_column_headers(hookup_dict['hookup'])
+            if state_args['show_levels']:
+                hookup_dict = self.__hookup_add_correlator_levels(
+                    hookup_dict, state_args['levels_testing'])
+        else:
```

Comment:
I think it's easier to reason about if/else statements if the short clause comes first, since when you're reading the code that keeps the `else` closer to the original `if` proposition. In this case that would mean:

```python
if not len(hookup_dict['hookup'].keys()):
    hookup_dict['columns'] = []
else:
    hookup_dict = self.__add_hookup_timing(hookup_dict)
    ...
```